components/inputs/input-time.js

```diff
@@ -461,9 +461,17 @@ class InputTime extends InputInlineHelpMixin(FocusMixin(LabelledMixin(SkeletonMi
 		// open and focus dropdown on down arrow or enter
 		if (e.keyCode === 40 || e.keyCode === 13) {
 			if (!this._dropdownFirstOpened) this._dropdownFirstOpened = true;
-			this.opened = true;
 			e.preventDefault();
+			await this._waitForItems();
```

Comment:
I was seeing consistent vdiff results where it wasn't focusing on the first item in the menu when the time input was opened. I tracked this down to the menu bailing on its focus code because there weren't any items and it therefore couldn't determine where to move focus. That's because time inputs don't render their menu items until the menu is first opened as a performance thing.

So this change sets `this._dropdownFirstOpened` to `true` which will cause a re-render and the items to be rendered. It then waits for them to be present in the DOM, and only then does it set `opened` which will cause another re-render and the menu code will take over to place focus.